orttraining/orttraining/test/training_ops/cpu/tensor/gather_nd_grad_op_test.cc

```diff
@@ -1,6 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// TODO(askhade): enable these tests for all training builds.
```

Comment:
Are tests failing at random? Shall we disable only those that are failing so we don't need to go looking for the failing ones when we work on fixing it?